docs/how-to-contribute.md

```diff
@@ -32,18 +32,16 @@ make IMG=test/operator:test deploy
 
 Note: `IMG` currently defaults to: `datadog/datadog-operator:latest`
 
-## \[TMP\] how to test `v2alpha`
+## Test using `v2alpha`
```

Comment:
should we rm this section header and merge the cert manager install cmd with the section above?

Reply:
Changed :+1: